Workflows/Architecture review.md

```diff
@@ -1,4 +1,4 @@
-## The reason behind it
+# The reasons behind it
 
 Before you start programming, your application should go through an architecture review.
```

Comment:
I wouldn't use "programming" here. It feels too narrow minded.

Projects are usually grown and the code is one component of the whole. I'd write this maybe as:

> An architecture review is required before the team starts implementing the system.